projects/core/src/features-config/feature-toggles/config/feature-toggles.ts

```diff
@@ -171,6 +171,12 @@ export interface FeatureTogglesInterface {
    * Determines whether the controls in the `CarouselComponent` are focusable and accessible from the keyboard.
    */
   a11yFocusableCarouselControls?: boolean;
+
+  /**
+   * Determines whether `watchAddEntryFailEvent` method is executed or not in `CartQuickOrderFormComponent` component.
+   * The `watchAddEntryFailEvent` method listens to CartAddEntryFailEvent event.
+   */
+  cartQuickOrderRemoveListenToFailEvent?: boolean;
```

Comment:
IMHO it's worth also mentioning the second-order consequence of this change, which is avoiding an unnecessary doubled error message.

```suggestion
  /**
   * In `CartQuickOrderFormComponent` it stops calling the deprecated method 
   * `watchAddEntryFailEvent()`, which listens to the `CartAddEntryFailEvent`.
   *
   * It avoids showing an unnecessary doubled error message on the failure
   * of adding to cart.
   */
  cartQuickOrderRemoveListenToFailEvent?: boolean;
```